### ISSUE TITLE
The Event Detail page Action Button should display the event status as the text, and disable for certain status's

### ISSUE THREAD

**Issue by zmelamed** [revelrylabs/bn-mobile-react#320]
connected to big-neon/bn-web#473

Event status's will be set on the backend as:
On Sale Soon
Purchase Tickets
Sold Out
Tickets at the Door
Free
Rescheduled
Cancelled
Off-sale
Ended
Code Required

If event status is one of the following, change the color of the action button to #8E8E93 and disable the click action:
Rescheduled
Cancelled
Off-sale
Ended

**Comment by Krakaw** [comment]
The status is defined in event.override_status the defined enum is https://big-neon.github.io/bn-api-node/enums/_interfaces_resources_event_interface_.overridestatus.html
Waiting on big-neon/bn-api#664 to be merged in and I'm busy with big-neon/bn-web#473 so it can be tested more easily.

**Comment by zmelamed** [comment]
This is now live and unblocked

**Comment by Mcscott142** [comment]
I don't see any sort of Action button in these non-purchase tickets instances.

**Comment by jwietelmann** [comment]
I was kind of itching to rework this code anyway. Gonna give it a stab.

**Comment by jwietelmann** [comment]
Never mind. There is code around this that seems to have been put in by @zmelamed after he said it was live and unblocked. So I'm going to assume he did what he wanted to be done and close this. If there are further changes requested, we can make another issue.
